sdk/spring/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Release History
 
+## 5.19.0-beta.1 (Unreleased)
+
+### Spring Cloud Azure Autoconfigure
+This section includes changes in `spring-cloud-azure-autoconfigure` module.
+
+#### Features Added
+- Registry a connection string provider for Event Hubs only when conn string is not empty; support to custom the connection string provider for Event Hubs. [#42919](https://github.com/Azure/azure-sdk-for-java/pull/42919).
```

Comment:
`Registry a connection string provider for Event Hubs only when conn string is not empty; support to custom the connection string provider` -> `Support customize StaticConnectionStringProvider<AzureServiceType.EventHubs> bean`.